src/index.js

```diff
@@ -25,6 +25,10 @@ if (__CONFIG__.css_file_path != undefined) {
 
 // Configuration
 const ConfigDefaults = {
+	title: 'TeskaLabs SeaCat Auth',
```

Comment:
@byewokko You sure about this default title?

Reply:
i think so. it is the official title used in the documentation for example. and it follows the way it is done for LogMan.io